CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contribute
+
+You can help by finding out problems on the website (QA), sending suggestions, and/or coding.
+
+## Reporting problems or suggestions
+
+Go through [the issues](https://github.com/MozillaIndia/blog/issues) and create a new issue if yours is not already added.
+
+## Adding code
+
+### Setting up developer tools
+
+The blog is built using [Jekyll](https://jekyllrb.com/). To set up a local workspace, here is what you should do.
+
+1. Fork and clone the repo as described in git workflow below.
+2. For OS specific instructions on how to install Jekyll, refer the official Jekyll [Docs](https://jekyllrb.com/docs/installation/).
+3. Run the blog on a localhost server using the following command at the root of the repo
+
+```
+bundle exec jekyll serve
+```
+
+(To know more about how Jekyll works, refer the official Jekyll [Step by Step tutorial](https://jekyllrb.com/docs/step-by-step/01-setup/).)
+
+4. By default the server runs at `localhost:4000`. For any changes to [_config.yml](_config.yml), the server must be restarted.
+
+
+### Git workflow
+
+- Fork this repo using the button at the top.
+- Clone your forked repo locally.
+
+```
+$ git clone git@github.com:yourname/blog.git
```

Comment:
Since we are suggesting SSH cloning, Might I suggest that you add a link to [Setting up SSH](https://help.github.com/en/articles/connecting-to-github-with-ssh) in the git workflow?

Reply:
Oh, I wasn't aware this is SSH cloning. I just took the content from the main Mozilla India website's [CONTRIBUTING.md](https://github.com/MozillaIndia/mozillaindia.github.io/blob/gatsby/CONTRIBUTING.md) as it is.

Should I replace the code with the one below?
```sh
$ git clone https://github.com/yourname/blog.git
```

Let's keep the setup simple for contributors.

Reply:
Hmm, Since it's Jekyll, I think most of the people will be developing with Linux; which till I know lacks a credential manager (I don't believe that caching the password can count as credential management) and thereby requiring people with 2FA to attach personal access token every time instead of the password. It would be better for people to clone with SSH. 

Then again, I doubt beginners will be aware of such things and/or use 2FA. So keeping it HTTPS sounds like a possible way too. Not sure about why the Mozilla India contributing.md recommends using SSH, but I guess what I stated above might be the reason. So let's go with HTTPS for now. You can switch all the SSH commands to the HTTPS version.

Reply:
Yeah I myself am on linux (Ubuntu) and have never used SSH for cloning a git repo. I tried running `git clone https://github.com/cseas/blog.git` and got the following error:

![image](https://user-images.githubusercontent.com/29686866/66820067-a02fe080-ef5d-11e9-9c67-60dbbcf1a750.png)

Anyway, I've changed the SSH commands to HTTPS as suggested.

Reply:
That's cause you need to setup SSH xD, feel free to hit me up on twitter, etc. if you are interested in the same. Changes look good to me!

Reply:
I had to leave a comment here. Unless you want beginners to stay that way, what we need to show them is the way to grow. And not take shortcuts.

Reply:
That's true! Shortcuts are never the way to go, hence my initial recommendation of adding the link to Setting up SSH. I will switch it back to SSH along with a proper guide for it mentioned initially in the next week once my exams are done. Meanwhile, @cseas can give it a tackle if interested, feel free to resolve any doubts!

Reply:
Updated #5 with the discussion here. I might not be able to work on it at the moment.